packages/quantic/force-app/examples/main/lwc/exampleQuanticCategoryFacet/exampleQuanticCategoryFacet.js

```diff
@@ -0,0 +1,77 @@
+import {api, LightningElement, track} from 'lwc';
+
+export default class ExampleQuanticFacet extends LightningElement {
+    @api engineId = 'quantic-category-facet-engine';
+    @track config = {};
+    isConfigured = false;
+
+    pageTitle = 'Quantic Category Facet';
+    pageDescription = 'The Quantic Category Facetis a list of values for a certain field occurring in the results, ordered using a configurable criterion.'
+    options = [
+        {
+            attribute: 'field',
+            label: 'Field',
+            description: 'The name of the field to display as a facet.',
+            defaultValue: 'geographicalhierarchy'
+        },
+        {
+            attribute: 'label',
+            label: 'Label',
+            description: 'The label to use as the facet title.',
+            defaultValue: 'Country'
+        },
+        {
+            attribute: 'basePath',
+            label: 'BasePath',
```

Comment:
I think sentence case is better for the labels. Please update other labels in this file as well.

`BasePath` --> `Base path`